.github/CODEOWNERS

```diff
@@ -913,7 +913,7 @@
 /**/*Management*/                                                  @archerzz @ArcturusZhang @ArthurMa1978
 
 # PRLabel: %Mgmt
-/**/Azure.ResourceManager*/                                        @archerzz @ArcturusZhang @ArthurMa1978
+/**/Azure.ResourceManager*/                                        @archerzz @ArcturusZhang @ArthurMa1978 @bicep-admins
 
 # -----------------------------------------------------------------
```

Comment:
Please add a dedicated section for this specific path and label below L927, preserving alphabetical ordering by tag.

Reply:
@anamikapan We need to add `@Azure/bicep-admins` as an owner for the `/sdk/resources/Azure.ResourceManager.Resources/` path. Please ensure that the team name is fully qualified to include the `Azure` organization.

Reply:
@anamikapan Let's use @azure/arm-sdk-owners instead. I just created it.